packages/@uppy/react/src/Dashboard.js

```diff
@@ -10,6 +11,11 @@ const h = React.createElement
  */
 
 class Dashboard extends React.Component {
+  constructor (props) {
+    super(props)
+    this.validProps = getHTMLProps(props)
```

Comment:
This only gathers the props from the initial render, which can be an issue if the user changes the props over time. Is `getHTMLProps` fast enough to just do it on every render?